base/src/commonMain/kotlin/flow/ColdFlowable.kt

```diff
@@ -36,7 +36,7 @@ import kotlinx.coroutines.sync.withLock
  * @param autoClose When set to `true` automatically closes the flow when all observers are removed. Defaults to `true`.
```

Comment:
Also remove this frm the doc

Reply:
41be0379a420a19c2ccc6635e56f9a9db2f99d8a